websites/views.py

```diff
@@ -40,3 +46,27 @@ def get_queryset(self):
         if website_type is not None:
             queryset = queryset.filter(type=website_type)
         return queryset
+
+
+class WebsiteStarterViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    viewsets.GenericViewSet,
+):
+    """
+    Viewset for WebsiteStarters
+    """
+
+    permission_classes = (ReadonlyPermission,)
```

Comment:
We don't have pagination classes here like we do for `WebsiteViewSet`. Is that intentional? I can see reasons for and against so no preference here

Reply:
Just an oversight. I'll add